code/game/objects/items/storage/toolbox.dm

```diff
@@ -447,3 +447,65 @@
 	desc = "A gun case. Has the symbol of the Third Soviet Union stamped on the side."
 	weapon_to_spawn = /obj/item/gun/ballistic/automatic/plastikov
 	extra_to_spawn = /obj/item/food/rationpack //sorry comrade, cannot get you more ammo, here, have lunch
+
+/obj/item/storage/toolbox/guncase/monkeycase
+	name = "monkey gun case"
+	desc = "Everything a monkey needs to truly go ape-shit. There's a paw-shaped hand scanner lock on the front of the case."
+
+/obj/item/storage/toolbox/guncase/monkeycase/Initialize(mapload)
+	. = ..()
+	atom_storage.locked = STORAGE_SOFT_LOCKED
+
+/obj/item/storage/toolbox/guncase/monkeycase/attack_self(mob/user, modifiers)
+	if(!monkey_check(user))
+		return
+	. = ..()
+
+/obj/item/storage/toolbox/guncase/monkeycase/attack_self_secondary(mob/user, modifiers)
+	attack_self(user, modifiers)
+	return
+
+/obj/item/storage/toolbox/guncase/monkeycase/attack_hand(mob/user, list/modifiers)
+	if(!monkey_check(user))
+		return
+	. = ..()
+
+/obj/item/storage/toolbox/guncase/monkeycase/proc/monkey_check(mob/user)
+	if(atom_storage.locked == STORAGE_NOT_LOCKED)
+		return TRUE
+
+	if(is_simian(user))
+		atom_storage.locked = STORAGE_NOT_LOCKED
+		to_chat(user, span_notice("You place your paw on the paw scanner, and hear a soft click as [src] unlocks!"))
+		playsound(src, 'sound/items/click.ogg', 25, TRUE)
+		return TRUE
+	to_chat(user, span_warning("You put your hand on the hand scanner, and it rejects it with an angry chimpanzee screech!"))
+	playsound(src, "sound/creatures/monkey/monkey_screech_[rand(1,7)].ogg", 75, TRUE)
+	return FALSE
+
+/obj/item/storage/toolbox/guncase/monkeycase/PopulateContents()
+	switch(rand(1, 100))
```

Comment:
why are you doing a 1-100 randomisation when you only have three equally likely outcomes?
It doesn't really _matter_ but it's a slightly confusing approach when `rand(1, 3)` would do 😅

Reply:
oooga boooga